solr/solr-ref-guide/src/coreadmin-api.adoc

```diff
@@ -50,6 +94,45 @@ The `CREATE` action creates a new core and registers it.
 If a Solr core with the given name already exists, it will continue to handle requests while the new core is initializing. When the new core is ready, it will take new requests and the old core will be unloaded.
 
 `admin/cores?action=CREATE&name=_core-name_&instanceDir=_path/to/dir_&config=solrconfig.xml&dataDir=data`
+[.dynamic-tabs]
+--
+[example.tab-pane#v1coreadmin-create]
+====
+[.tab-label]*V1 API*
+
+Assuming you are using an existing configSet to create your new core:
+[source,bash]
+----
+http://localhost:8983/solr/admin/cores?action=CREATE&&name=techproducts_v2&configSet=sample_techproducts_configs
+
+----
+
+If you have already existing core files deployed on disk, and need to just create the Solr core from them then the url will look something like
+[source,bash]
+----
+admin/cores?action=CREATE&name=_core-name_&instanceDir=_path/to/dir_&config=solrconfig.xml&dataDir=data
+----
+====
+
+[example.tab-pane#v2coreadmin-create]
+====
+[.tab-label]*V2 API*
+
+Assumes SOLR-15372 is committed.
```

Comment:
[0] The way I read SOLR-15372, it's specific to configuration used by our techproducts example. Is that right?

I know we mention that these commands assume the "techproducts" example, but it still seems weird to mention this bug here.  Only 1% of readers are going to be "following along " with a techproducts example locally, and I'd hate for users to think that there's some bug with core-creation generally.

Reply:
I agree with Jason. The Jira is only relevant to the example you've chosen to use, not the API, so the bug is completely irrelevant to the overall documentation of the API.

Reply:
I've tried a couple of workarounds, and no joy, so just going to take out the comment, and leave it...